packages/cashier/src/Stores/account-transfer-store.js

```diff
@@ -99,6 +100,11 @@ export default class AccountTransferStore {
         this.selected_to.balance = balance;
     }
 
+    @action.bound
+    setOnRemount(func) {
+        this.onRemount = func;
+    }
+
```

Comment:
As advised, no need to declare this here. You need to call `setOnRemount` from `general-store`.